### PR TITLE
minor journey contact.njk 'opens in new tab' moved within anchor tags

### PR DESCRIPTION
**Pull request by neil-stephen-mcgonigle**

### CODE REVIEW

**Review by BarryPiccinni — APPROVED** [review]

File: partials/contact.njk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% from "components/table/macro.njk" import govukTable %}
 <p class="govuk-body govuk-!-margin-bottom-1">Telephone: 0300 003 3601</p>
 <p class="govuk-body govuk-!-margin-bottom-1">Outside the UK: +44 (0) 203 684 2517</p>
-<p class="govuk-body"><a href="https://www.relayuk.bt.com" class="govuk-link">Relay UK</a>: (if you cannot hear or speak on the phone) 18001 0300 003 3601</p>
+<p class="govuk-body"><a href="https://www.relayuk.bt.com" class="govuk-link">Relay UK(opens in new tab)</a>: (if you cannot hear or speak on the phone) 18001 0300 003 3601</p>
```
Comment (armoj) [inline partials/contact.njk:4]: doesn't have the `target="_blank"` for it to open in a new tab

File: partials/contact.njk [hunk 2]
```diff
@@ -56,4 +56,4 @@
             ]
         ]
     }) }}
-<p class="govuk-body"><a href="https://www.gov.uk/call-charges" class="govuk-link">Find out about call charges</a></p>
+<p class="govuk-body"><a href="https://www.gov.uk/call-charges" class="govuk-link">Find out about call charges(opens in new tab)</a></p>
```
Comment (armoj) [inline partials/contact.njk:59]: doesn't have the target="_blank" for it to open in a new tab

**Review by armoj — APPROVED** [review]

File: partials/contact.njk (hunk 1 above)
Comment (neil-stephen-mcgonigle) [reply partials/contact.njk:4]: doh!

**Review by armoj — APPROVED** [review]

File: partials/contact.njk (hunk 2 above)
Comment (neil-stephen-mcgonigle) [reply partials/contact.njk]: you know I have just tested this, aye I know I should have tested it first... it's not opening in a new tab.. 
Seems to be an issue across the app, 
image
hmmm

**Pull request merged**